contrib/manifests/cert-manager/with-rbac.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
   namespace: "cert-manager"
   labels:
     app: cert-manager
-    chart: cert-manager-v0.3.2
+    chart: cert-manager-v0.4.0-dev.0
```

Comment:
:question: Isn't there a macro we can use here (and below)? Like `{{ .Chart.Version }}` or something?

Reply:
Does helm update the labels on the resources when you perform a `helm upgrade` ?

Reply:
Yes, and it is used. You are looking at auto-generated manifests here 😄 (for use when not using Helm)